app/components/combined_system_message/last_users.test.js

```diff
@@ -1,45 +0,0 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
-// See LICENSE.txt for license information.
-
-import React from 'react';
-import {configure} from 'enzyme';
-import Adapter from 'enzyme-adapter-react-16';
-configure({adapter: new Adapter()});
-
-import {Posts} from 'mattermost-redux/constants';
-
-import {shallowWithIntl} from 'test/intl-test-helper';
-
-import LastUsers from './last_users';
-
-describe('LastUsers', () => {
-    const baseProps = {
-        actor: 'actor',
-        expandedLocale: {id: 'expanded_locale_id', defaultMessage: 'Expanded Locale'},
-        postType: Posts.POST_TYPES.ADD_TO_TEAM,
-        style: {activityType: {fontSize: 14}, link: 1, text: {opacity: 0.6}},
-        userDisplayNames: ['User One', 'User Two'],
-    };
-
-    test('should match snapshot', () => {
-        const wrapper = shallowWithIntl(
-            <LastUsers {...baseProps}/>
-        );
-
-        const expanded = wrapper.instance().renderExpandedView(baseProps.expandedLocale, baseProps.userDisplayNames, baseProps.actor, 1, baseProps.style);
-        expect(expanded).toMatchSnapshot();
-
-        const collapsed = wrapper.instance().renderCollapsedView(baseProps.postType, baseProps.userDisplayNames, baseProps.actor, 1, baseProps.style);
-        expect(collapsed).toMatchSnapshot();
-    });
-
-    test('should match state on handleOnClick', () => {
-        const wrapper = shallowWithIntl(
-            <LastUsers {...baseProps}/>
-        );
-
-        wrapper.setState({expand: false});
-        wrapper.instance().handleOnClick({preventDefault: jest.fn()});
-        expect(wrapper.state('expand')).toEqual(true);
-    });
-});
```

Comment:
Had to remove tests both on `CombinedSystemMessage` and `LastUsers`.  It might be due to babel/jest transpiling issue of package/s being by these components.  Unfortunately, I already spent significant time to sort it out but to no avail.

Reply:
Failing test is:
```
    /mattermost/mattermost-mobile/node_modules/jail-monkey/index.js:12
    export default {
    ^^^^^^
    
    SyntaxError: Unexpected token export

    > 1 | // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
      2 | // See LICENSE.txt for license information.
      3 | import {NativeModules, DeviceEventEmitter} from 'react-native';
      4 | import LocalAuth from 'react-native-local-auth';
      
      at ScriptTransformer._transformAndBuildScript (node_modules/jest-runtime/build/script_transformer.js:316:17)
      at Object.<anonymous> (app/mattermost_managed/mattermost-managed.ios.js:1:353)
      at Object.<anonymous> (app/mattermost_managed/index.js:1:170)
```

Reply:
Removal is temporary.  I'll tackle this separately.